hit_lc_proxy.bash

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# This script allows running against a specific lantern-cloud proxy identified by the PFE IP
+# It assumes that lantern-cloud is located on the filesystem as a sibling of lantern-desktop
```

Comment:
I think this assumption may no longer be valid (I don't know, maybe everyone else has all of their Lantern repos in the same folder?  I only have the Go ones in the same folder).

We could add a variable for this:
```
# Assume lantern-cloud is a sibling of lantern-client unless told otherwise.
LANTERN_CLOUD=$([[ ! -z "$LANTERN_CLOUD" ]] && echo "$LANTERN_CLOUD" || echo "../lantern-cloud")
```

then use it below, like:
```
$LANTERN_CLOUD/bin/ptool ...
```

We would need to get rid of the `-u` flag on `set` (line 6), but I'm not sure why we'd need that anyway.  This is a pretty simple script and bugs due to unset variables should be easy to trace.

Reply:
Thanks.. I basically just copied the script from lantern-desktop and didn't look at the comment. I do have all my Lantern repos in the same folder but I made the change you suggested to use a variable for lantern-cloud..

Reply:
Oh yeah no worries, I figured that was the case.  Cool, thanks!